### DISCUSSION TITLE
warning IDE0032

### DISCUSSION THREAD

**Discussion by Oreosenseii** [winstxnhdw/lc-hax#570] [General]
lc-hax\lc-hax\Scripts\Modules\Possession\Possession.cs(2,5): warning IDE0032: Use auto property (https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0032)
![image](https://github.com/user-attachments/assets/d855f775-37f3-4f4c-bde0-b8838dcceff4)

- how can i fix this thanks <3

**Comment by winstxnhdw** [comment]
Yeah, this is an issue on my end. I'll have to fix this. Not sure why it started being a problem now after so long.

**Reply by Oreosenseii** [reply]
owww... i hope you can fix it .

**Reply by winstxnhdw** [reply]
It shouldn't cause any issue though?

**Reply by Oreosenseii** [reply]
i can't use

**Reply by winstxnhdw** [reply]
The actual error you are having is "access is denied"

**Reply by Oreosenseii** [reply]
how can i access it?

**Reply by winstxnhdw** [reply]
See #513

**Reply by winstxnhdw** [reply]
Are you running Steam as admin?
